shared/settings/notifications/index.js

```diff
@@ -91,6 +94,19 @@ const Notifications = (props: Props) =>
             unsubscribedFromAll={props.groups.app_push.unsubscribedFromAll}
           />}
 
+        {props.groups.security &&
+          props.groups.security.settings &&
+          <Group
+            allowEdit={props.allowEdit}
+            groupName="security"
+            onToggle={props.onToggle}
+            onToggleUnsubscribeAll={() => {}}
```

Comment:
the group component should have this be optional and do a check before calling vs passing in a noop